src/main/java/com/example/SteDziPanki/repository/TripRepository.java

```diff
@@ -14,6 +13,9 @@
 
 
     Trip save(Trip entity);
-   List<Trip> findAll();
-   Optional<Trip> findByAllInclusive(String status);
+
+    List<Trip> findAll();
```

Comment:
któryś raz to widzę, nie musicie nadpisywać metod które już są w tym interfejsie.

Reply:
Prosze Sprawdz jeszcze raz

Reply:
bo troche doszło

Reply:
o tej metodzie chodzi o to że nie trzeba nadpisywać findAll, save itd. To są bazowe metody z JpaRepository

Reply:
jeśli nie robi to problemów to zostawił bym to ponieważ wizualnie widzę ze jest to wyciągane z tego interfejsu a nie pojawia sie z znikąd mimo ze wiem o tym ze są to metody podbierane z JpaRepository